src/workflow/restore.c

```diff
@@ -44,6 +46,13 @@ bool workflow_restore_backup(const RestoreBackupRequest* restore_request)
         workflow_status_create("Could not\nrestore backup", false);
         return false;
     }
+    if (!workflow_confirm_time(
+            restore_request->timestamp, restore_request->timezone_offset, true)) {
+        return false;
+    }
+    if (!securechip_u2f_counter_set(restore_request->timestamp)) {
```

Comment:
in securechip.c, disable the two u2f functions for btc-only

Reply:
ack